### DISCUSSION TITLE
Is it possible to style chips in QSelect?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#13200] [General - Components / Directives / etc] [answered]
I would like to display the selected values from QSelected as chips so I use the `use-chips` option.

The chips are however all gray - I would like to style them in accordance with the color I store with each value (in addition to what is displayed I also keep track of its color).

Is there a way to style each chip in QSelect separately?

**Comment by MilosPaunovic** [comment, marked as answer]
Absolutely, with the help of slots.
[https://quasar.dev/vue-components/select#example--selected-item-slot](https://quasar.dev/vue-components/select#example--selected-item-slot)

**Reply by wsw70** [reply]
Ah yes, thank you.

Would you know how to extract information from the selected item? I use

```
   <q-select
      v-model="chosenTags"
      :options="possibleTags"
      label="edit tags"
      multiple
      use-chips
      use-input
    >
      <template v-slot:selected-item="scope">
        <q-chip
          removable
          @remove="scope.removeAtIndex(scope.index)"
          :tabindex="scope.tabindex"
          :color="scope.opt.color"
          text-color="black"
          class="q-ma-xs"
          size="md"
        >
          {{ scope.opt.label }}
        </q-chip>
      </template>
    </q-select>
```

My model is a 

     Array<{label: string, value: {Name: string, Color: string}, color: string}>

The documentation for `opt` in the `selected-item` scope states that 

> Selected option -- its value is taken from model

I therefore expected that `:color="scope.opt.color"` would use the value of `color` in my data - but it (visually) does not. Is this the right approach (just checking before jumping into debug, to make sure I did not get all of this completely wrong)

**Reply by wsw70** [reply]
I found a solution, apparently `:color` is not the way to go:

    :style="{backgroundColor: scope.opt.color}"
